### ISSUE TITLE
OrderbookLevelV5 should be a pair of strings

### ISSUE THREAD

**Issue by nullobject** [tiagosiebler/bybit-api#315]
The `OrderbookLevelV5` type looks like this:

```ts
export default interface OrderbookLevelV5 {
  price: string;
  size: string;
}
export interface OrderbookResponseV5 {
  s: string;
  b: OrderbookLevelV5[];
  a: OrderbookLevelV5[];
  ts: number;
  u: number;
}
```

but it does not match the Bybit [Get Orderbook](https://bybit-exchange.github.io/docs/v5/market/orderbook) API definition:

```
{
    "retCode": 0,
    "retMsg": "OK",
    "result": {
        "s": "BTCUSDT",
        "a": [
            [
                "16638.64",
                "0.008479"
            ]
        ],
        "b": [
            [
                "16638.27",
                "0.305749"
            ]
        ],
        "ts": 1672765737733,
        "u": 5277055
    },
    "retExtInfo": {},
    "time": 1672765737734
}
```

The `OrderbookLevelV5` type should be a pair of strings:

```ts
export type OrderbookLevelV5 = [string, string];
```

**Comment by tiagosiebler** [comment]
Based on what you found, you're absolutely right. Let me just double check that this type isn't used elsewhere (whether we can update it or if we need a new type for this particular endpoint).

**Comment by tiagosiebler** [comment]
Will be fixed in v3.9.2 as soon as #316 merges. Thanks for reporting this.

**Comment by tiagosiebler** [comment]
Fixed via #316

**Comment by nullobject** [comment]
Thanks for fixing this @tiagosiebler.
